src/main/app/src/app/shared/edit/edit-datum-groep/edit-datum-groep.component.html

```diff
@@ -73,6 +73,11 @@
                     <mat-icon class="play-button">pause_circle_outline</mat-icon>
                 </button>
             </div>
+            <div *ngIf="hervat" class="play-control">
```

Comment:
Waarom is hier een tweede button voor nodig? De logica dat je niet kunt opschorten staat al op regel 150 in zaak-view-component.html

Er moet alleen onderscheid gemaakt worden in de 2 redenen waarom de "opschorten" knop gedisabled is. Want deze twee redenen geven een verschillende melding als je er overheen hoovert:
1. De zaak is opgeschort met reden
2. De zaak is ooit opgeschort met reden.

de boodscap is bijna hetzelfde, Of de zaak is opgeschort of is opgheschort geweest kun je bepalen via de "opgeschort" boolean van dit component

Reply:
Ik heb hier expres een aparte button van gemaakt omdat er behoorlijk wat gedrag anders moet zijn. De button mag niet meer van icon veranderen on-hover, de title moet anders zijn en de button moet zeker zichtbaar zijn. Het gedrag voor het niet kunnen opschorten van een zaak vanwege hervatten (disabled button) en vanwege andere redenen (geen button aanwezig) is volledig anders. Dat wilde ik niet in &eacute;&eacute;n button proppen om de leesbaarheid van de front-end code ten goede te komen.

Dat hier een aparte button voor is met ander gedrag dan het 'normale' verbergen van de opschorten-knop, en dat de knop in andere gevallen nog wel verstopt moet worden is afgestemd met Roy.

Reply:
Besproken via teams - samengevoegd in opvolgende commit.